docs/api-reference/decorators.md

```diff
@@ -137,6 +137,41 @@ DBOS supports declaration of the following values for `IsolationLevel`:
 
 The transaction semantics of these levels are defined for PostgreSQL [here](https://www.postgresql.org/docs/current/transaction-iso.html).
 
+#### `@StoredProcedure`
+Registers a function as a [DBOS stored procedure](../tutorials/stored-proc-tutorial.md).
+
+The first argument of the decorated function must be a [`StoredProcedureContext`](contexts#storedprocedurecontext), which provides access to the database.
+
+```typescript
+@StoredProcedure({readOnly: true})
+static async doLogin(ctx: StoredProcedureContext, username: string, ) {
+  ...
+}
+```
+
+`@StoredProcedure()` takes an optional `StoredProcedureConfig` object:
+
+```typescript
+interface StoredProcedureConfig {
+  isolationLevel?: IsolationLevel;
+  readOnly?: boolean;
+  executeLocally?: boolean
+} 
+```
+
+The `readOnly` and `isolationLevel` config fields behave the same as their [@Transaction config](contexts#transactioncontextt) counterparts.
+
+The `executeLocally` config field can be used to control where the logic of a stored procedure is executed.
```

Comment:
Maybe explain a bit more on what `executeLocally` means. Because "local" can mean being local to the database or to the application runtime.

Especially, we can explain that when `executeLocally` is false, DBOS Transact invokes the function as a stored procedure in the application's database.